inst/builtin/bs5/shiny/tables/_rules.scss

```diff
@@ -0,0 +1,87 @@
+.table.dataTable {
```

Comment:
Let's add a comment to the top about how these rules could make sense to tack onto `card.scss` someday, but at least for now, we're only going to commit to supporting this with the preset